examples/private_set_intersection/example.toml

```diff
@@ -16,10 +16,11 @@ additional_args = [
 
 [clients]
 rust = { Cargo = { cargo_manifest = "examples/private_set_intersection/client/rust/Cargo.toml" }, additional_args = [
+  "--set-id=rust",
```

Comment:
Perhaps add a comment somewhere around here explaining why these have different ids, and what they mean, etc.